src/dyndep.cc

```diff
@@ -43,9 +43,7 @@ bool DyndepLoader::LoadDyndeps(Node* node, DyndepFile* ddf,
 
   // Update each edge that specified this node as its dyndep binding.
   std::vector<Edge*> const& out_edges = node->out_edges();
-  for (std::vector<Edge*>::const_iterator oe = out_edges.begin();
-       oe != out_edges.end(); ++oe) {
-    Edge* const edge = *oe;
+  for (Edge* edge : out_edges) {
```

Comment:
nit: use `const Edge*` here. Same concern for other loops below.

Reply:
Your proposal does not work, `map->find` complains no match as it "would lose const qualifier".

Reply:
Weird, what did the original const_iterator do then?

Reply:
Oh I see, this was doing the equivalent of `Edge* const&`, so using `Edge*` here is equivalent. Let's keep your code though, sorry for the noise :)